.changeset/smart-foxes-sit.md

```
@@ -0,0 +1,5 @@
+---
+"extension": patch
+---
+
+Fix MCP not connecting to new Taskmaster version
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)